beluga_ros/test/cmake/catkin.cmake

```diff
@@ -55,3 +55,10 @@ target_link_libraries(
   ${PROJECT_NAME}
   ${catkin_LIBRARIES}
   gmock_main)
+
+catkin_add_gmock(test_point_cloud test_point_cloud.cpp)
+target_link_libraries(
+  test_point_cloud
+  ${PROJECT_NAME}
+  ${catkin_LIBRARIES}
+  gmock_main)
```

Comment:
@pvela2017 nit: missing newline.

Reply:
Where should the new line go?

Reply:
Fixed 0daac06440f48536bba7122e5cf768aac9471ecd